packages/swc-angular-plugin/src/component_decorator_visitor.rs

```diff
@@ -96,10 +98,11 @@ impl VisitMut for ComponentDecoratorVisitor {
 
         if key.sym.eq("styleUrls") {
             node.key = PropName::Ident(Ident {
+                ctxt: SyntaxContext::default(),
```

Comment:
i'm not sure what should be used here, but `default` works. Previous version of Ident does not have that required field.

Reply:
weird but default lgtm